editor/test/integration/model_utility_test.clj

```diff
@@ -0,0 +1,70 @@
+;; Copyright 2020 The Defold Foundation
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;; 
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;; 
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
+(ns integration.model-utility-test
+  (:require [clojure.set :as set]
+            [clojure.test :refer :all]
+            [clojure.java.io :as io]
+            [editor.model-loader :as model-loader]
+            [editor.workspace :as workspace]
+            [integration.test-util :as test-util]))
+
+(defn- sequence-buffer-in-mesh [mesh data-stride data-key index-key]
+  (let [partitioned-data (vec (partition data-stride (mesh data-key)))
+        sequenced-data (into [] (mapcat #(partitioned-data %) (mesh index-key)))
```

Comment:
`#(partitioned-data %)` could be simply `partitioned-data` here - vectors are functions that take an index.